courses/serializers.py

```diff
@@ -27,12 +29,23 @@ def get_programpage_url(self, program):
         except ProgramPage.DoesNotExist:
             return
 
+    def get_enrolled(self, program):
+        """
+        Returns true if the user
+        """
+        if 'request' in self.context:
```

Comment:
understood. i think it's just coincidental that we only use it from views that require a logged-in user, but this is pretty low-risk